src/components/morphine-ui/DataDisplay/Card/ProductCard/VerticalProductCard/VerticalProductCard.css

```diff
@@ -0,0 +1,30 @@
+.product-card {
+  max-width: 17rem;
+}
+.product-card:hover {
+  box-shadow: 5px 10px 18px #888888;
+}
+.product-cardPosition {
+  position: relative;
+}
+.product__image {
+  height: auto;
+  max-width: 100%;
+  object-fit: cover;
+  margin: 0 auto;
+}
+.product__content {
+  width: 100%;
+}
+.label--sale {
+  background-color: var(--themeRed);
+  color: var(--light);
+  transform: rotate(270deg);
+}
+.label--salePosition {
+  position: absolute;
+  top: var(--space-lg);
+  /* top: 0; */
```

Comment:
you can remove commented code if you are not going to use it again

Reply:
done